TramsDataApi/DatabaseModels/TramsDbContext.cs

```diff
@@ -1,7 +1,9 @@
 using Microsoft.EntityFrameworkCore;
 using System;
 using System.Linq;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
I couldn't see why this namespace was added? X509Certificates

Reply:
I don't know either. But I've removed it.